.bumpversion.cfg

```diff
@@ -1,10 +1,10 @@
 [bumpversion]
-current_version = 2.0.0.dev3
+current_version = 2.0.0
 commit = False
 tag = False
-parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?:\.dev(?P<dev>\d+))?
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<release>[a-z]+)(?P<dev>\d+))?
 serialize = 
-	{major}.{minor}.{patch}.dev{dev}
+	{major}.{minor}.{patch}.{release}{dev}
```

Comment:
what is the `release` used for?

Reply:
The idea was that we would be able to run `bumpversion release` and get rid of the `.devN` label. If you take a look at line 10 and below, you can see that by default each version will be followed by release label equal to `dev`. Then, when we run `bumpversion release`, it will change the release label to `prod`, which is an optional value, so it won't be displayed in the version.